iib/web/migrations/versions/daf67ddcf4a1_add_support_for_graph_update_mode_in_.py

```diff
@@ -0,0 +1,25 @@
+"""Add support for graph_update_mode in Add endpoint.
+
+Revision ID: daf67ddcf4a1
+Revises: 8d50f82f0be9
+Create Date: 2023-07-27 15:37:59.568914
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = 'daf67ddcf4a1'
+down_revision = '8d50f82f0be9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table('request_add', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('graph_update_mode', sa.String(), nullable=True))
```

Comment:
I did think about it but then we would have to keep a close eye on if OPM changes something in their code related to the `--mode` option. And maybe OPM's default value is not something we want to rely our code on. 

Also, the parameter is restrictive and if we set it to a default, it may be confusing/misleading for the end user to see a value appear out of nowhere in the response when they don't submit anything. Hence, if you agree with this, I would like to recommend keeping it as is.

Reply:
That makes sense. Let's keep it the current way.